qlib/tests/data.py

```diff
@@ -140,6 +144,9 @@ def qlib_data(
         -------
 
         """
+        if exists_skip and exists_qlib_data(target_dir):
+            return
+
```

Comment:
It will better to keep the default behavior consistent with the previous version for compatible reason.